setup.cfg

```diff
@@ -16,6 +16,7 @@ show-response = 1
 minversion = 3.0
 python_files = test_?*.py
 norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "astropy_helpers"
+remote_data_strict = True
```

Comment:
What does this mean?

Reply:
It disables internet when remote data is not specified